### PR TITLE
Feature/mapbox

### PR DESCRIPTION
**Pull request by CamdenBoBamden**
**Thanks for updating the default 

Here's all the changes in one big pull request. Moving forward I'm going to try to align pull requests with cards in trello in smaller changes or features.

Outline of Changes:

Added mapbox for mapping function (free api key versus google maps)
Added markers from 3 separate json files that contain hard coded entries for 3 primary categories Food/Shelter/Health
Changes to CSS just to make this look a bit better

If you "checkout" this branch in VSCode you will need to do another npm install I believe for new dependencies and then npm start as usual to view through local host.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,55 +1,129 @@
...
 
 
 class App extends Component {
-  
   render(){
-    const foodBanks = data
     const { isGeolocationAvailable, isGeolocationEnabled, coords } = this.props;
+    const Map = ReactMapboxGl({
+      accessToken:
+      'pk.eyJ1IjoiY29saW5waGlsbGlwczY3IiwiYSI6ImNrMGZvd2MyaDAxdG8zbHJ6MmFnNnZpaTUifQ.tJ63Dty8f2DWdS1jenXPCA'
```
Comment (JustinKSU) [inline src/App.js:20]: Generally we don't want to share this?

File: src/App.js [hunk 2]
```diff
@@ -1,55 +1,129 @@
...
+
     return !isGeolocationAvailable ? (
       <div>Your browser does not support Geolocation</div>
   ) : !isGeolocationEnabled ? (
       <div>Geolocation is not enabled</div>
   ) : coords ? (
        <div className="App">
          
-        <h1>Social Services WebApp</h1>
+        <h1 className="title">Social Services WebApp</h1>
```
Comment (JustinKSU) [inline src/App.js:35]: Good use of a style.

File: src/App.js [hunk 3]
```diff
@@ -1,55 +1,129 @@
...
+            containerStyle={{
+            height: '80vh',
+            width: '90vw'
+          }}
+          >            
+            
+          <ZoomControl/>
+          
+          <Marker
+          coordinates={[-94.6359718, 38.919853200000006]}
```
Comment (JustinKSU) [inline src/App.js]: Why is this hard coded?

**Review by JustinKSU — APPROVED** [review]
Nice changes.  Do we need to protect the API key.

**Comment by ReclaimCamden** [comment]
That’s a good point. I’d think yes but even if we didn’t I’d still like to find out how to protect it. 

Camden Coulter
573-397-8143

**Comment by JustinKSU** [comment]
I would delete the comment with phone numbers

File: src/App.js (hunk 3 above)
Comment (CamdenBoBamden) [reply src/App.js]: Good catch I updated the code and I think it has been updated to below

```suggestion
         coordinates={[coords.longitude, coords.latitude]}
```

**Pull request merged**